helm/trino-gateway/values.yaml

```
@@ -37,6 +37,12 @@ config:
     serverConfig:
         node.environment: test
         http-server.http.port: 8080
+    dataStore:
+        #This stores the URLs of backend Trino servers and query history
```

Maybe

```suggestion
        # The connection details for the backend database for Trino Gateway and Trino query history
```

or just

        # The connection details for the backend database for Trino Gateway